lib/Views/Krankmeldung.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:flutter/material.dart';
+import '../api/api.dart';
+import '../app_type/app_type_managment.dart';
+import '../main.dart';
+
+class KrankmeldungWidget extends StatefulWidget {
+  State<KrankmeldungWidget> createState() => new KrankmeldungState();
+}
+
+class KrankmeldungState extends State<KrankmeldungWidget> {
+
+  String sek = '', name = '', grade = '', klasse = '', leader = '', email = '', remarks = '', info = '';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+            title: Text("Krankmeldung erstellen")
+        ),
+        body: Container(
+            margin: EdgeInsets.all(20),
+            child: SingleChildScrollView(
+              child: Column(
+                  children: [
+                    DropdownButton<String>(
+                        value: sek,
+                        onChanged: (newSek) {
+                          setState(() {
+                            sek = newSek;
+                          });
+                        },
+                        items: [
+                          DropdownMenuItem(
+                            child: Text('Sekundarstufe *'),
+                            value: '',
+                          ),
+                          DropdownMenuItem(
+                            child: Text('I (5. - 9. Klasse)'),
+                            value: 'sek1',
+                          ),
+                          DropdownMenuItem(
+                            child: Text('II (EF - Q2)'),
+                            value: 'sek2',
+                          )
+                        ]
+                    ),
+                    TextFormField(
+                        decoration: InputDecoration(labelText: "Name *"),
+                        maxLines: 1,
+                        initialValue: '',
+                        onChanged: (value) {
+                          name = value;
+                        }
```

Comment:
Ich bin mir nicht ganz sicher, da ich sowas mit nem Controller machen würde.
Aber müsste da nicht `initialValue: value` stehen?

Reply:
Das ist halt nur die startvalue. Ich will ja nicht am anfang irgendwas drin stehen haben, daher einfach ein leerer String. das beeinflusst nicht was später dann in value steht

Reply:
Ich bin mir nicht sicher wie das ist wenn Flutter den View neu baut